torch/distributed/_composable/fsdp/fully_shard.py

```diff
@@ -24,7 +24,7 @@
 
 # The decorator adds a state object to `module` that can be accessed via
 # `fully_shard.state(module)`. The state object and module are 1:1.
-@contract(state_cls=FSDPState)
+@contract(state_cls=FSDPState)  # type: ignore[operator]
```

Comment:
Curious: why did we need to add this `# type: ignore[operator]`? Were there mypy updates for this file?